### PR TITLE
Add Community Code Of Conduct

### PR DESCRIPTION
**Pull request by mathewcohle**
@VojtechBartos can u please add me to the org so I do not have to open PRs from fork please? :)

### CODE REVIEW

File: pages/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+BuildersLeaders Code of Conduct
```
Comment (mathewcohle) [inline pages/code-of-conduct.md:1]: @vire I will need a little guidance how to add this to the landing page as ref :)

File: pages/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+
+If you're not sure, ask someone instead of assuming.
+
+Community members are expected to not abuse this community to head-hunt and hire other community member and also not use it as a place to look for a job opportunity.
+
+Consequences And Enforcement
+----------------------------
+
+Violations of the Code of Conduct may be reported by contacting the team via sending an email to ADD-EMAIL-HERE. All reports will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.
```
Comment (mathewcohle) [inline pages/code-of-conduct.md:20]: @fedorocko do we have some org mail already?

File: pages/code-of-conduct.md (hunk 1 above)
Comment (vire) [reply pages/code-of-conduct.md:1]: What I would do for now - create a `<a href="full-url" >CODE OF CONDUCT</a>` to the CoC here on github and add logic to render the .md -> http://geekhmer.github.io/blog/2018/03/29/import-markdown-files-and-serve-its-content-in-next-dot-js/

**Review by vire — APPROVED** [review]
LGTM

**Review by vire — APPROVED** [review]
👍

File: pages/code-of-conduct.md (hunk 1 above)
Comment (mathewcohle) [reply pages/code-of-conduct.md]: After several unsuccessful attempts I give up and added simple ref to github (which will render it for us) :)

Should work for now, let's improve in the future

**Comment by VojtechBartos** [comment]
@mathewcohle unfortunately not admin here :) ask @vire or @fedorocko :)

**Comment by vire** [comment]
@VojtechBartos @mathewcohle you both have now rights ✊

**Comment by VojtechBartos** [comment]
Dankeshen!

File: pages/code-of-conduct.md (hunk 2 above)
Comment (mathewcohle) [reply pages/code-of-conduct.md]: let's add it later :)

**Pull request merged**